ntb_hw_switchtec.c

```diff
@@ -726,7 +726,8 @@ static int switchtec_ntb_peer_db_addr(struct ntb_dev *ntb,
 	offset += sndev->db_shift / 8;
 
 	if (db_addr)
-		*db_addr = pci_resource_start(ntb->pdev, 0) + offset;
+		*db_addr = pci_resource_start(ntb->pdev, 0) + offset +
+			  SWITCHTEC_GAS_MRPC_SIZE;
```

Comment:
Better to just add 

`offset += SWITCHTEC_GAS_TOP_CFG_OFFSET; `

above instead of adding to this line.